scripts/k8s/appsignals/sample-app/mongodb/mongodb-storageclass.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ebs-sc
```

Comment:
Does it have the same name as the one for python?

Reply:
Ah, these are for the k8s deploy, we didn't change those when we debugged this morning. I'll change now.